cloudify_cli/commands/instances.py

```diff
@@ -0,0 +1,78 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+"""
+Handles all commands that start with 'cfy instances'
```

Comment:
I'm wondering whether "cfy instances" is not a confusing term - perhaps it'd be better as "cfy node-instances"?
instances is obviously "better" in the sense that its more concise etc., but might be a confusing term overall..
I think I'd actually use "cfy instances" too - but i'd also consult others tomorrow about this specific part. API's important :)